Nodejs/Product/Nodejs/Options/NodejsIntellisenseOptionsControl.resx

```diff
@@ -322,6 +322,51 @@
   <data name="tableLayoutPanel2.ColumnCount" type="System.Int32, mscorlib">
     <value>2</value>
   </data>
+  <data name="_mediumIntelliSenseRadioButton.Anchor" type="System.Windows.Forms.AnchorStyles, System.Windows.Forms">
+    <value>Left</value>
+  </data>
+  <data name="_mediumIntelliSenseRadioButton.AutoSize" type="System.Boolean, mscorlib">
+    <value>True</value>
+  </data>
+  <data name="_mediumIntelliSenseRadioButton.CheckAlign" type="System.Drawing.ContentAlignment, System.Drawing">
+    <value>TopLeft</value>
+  </data>
+  <data name="_mediumIntelliSenseRadioButton.ImeMode" type="System.Windows.Forms.ImeMode, System.Windows.Forms">
+    <value>NoControl</value>
+  </data>
+  <data name="_mediumIntelliSenseRadioButton.Location" type="System.Drawing.Point, System.Drawing">
+    <value>8, 33</value>
+  </data>
+  <data name="_mediumIntelliSenseRadioButton.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
+    <value>8, 4, 8, 4</value>
+  </data>
+  <data name="_mediumIntelliSenseRadioButton.Size" type="System.Drawing.Size, System.Drawing">
+    <value>154, 21</value>
+  </data>
+  <data name="_mediumIntelliSenseRadioButton.TabIndex" type="System.Int32, mscorlib">
+    <value>5</value>
+  </data>
+  <data name="_mediumIntelliSenseRadioButton.Text" xml:space="preserve">
+    <value>&amp;Medium IntelliSense</value>
+  </data>
+  <data name="_mediumIntelliSenseRadioButton.TextAlign" type="System.Drawing.ContentAlignment, System.Drawing">
+    <value>TopLeft</value>
+  </data>
+  <data name="_mediumIntelliSenseRadioButton.ToolTip" xml:space="preserve">
+    <value>Medium completions (less memory/CPU usage)</value>
+  </data>
```

Comment:
Minor language suggestions, since I'm just now realizing that our label for "high" isn't actually "high", and we don't have a "low", so "medium" doesn't work as well. 
* Name: Quick IntelliSense
* Tooltip: Limited completions (reduced memory/CPU usage)